### PR TITLE
Add stromshield endpoint security to parsers

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
- Add stromshield endpoint security format

### CODE REVIEW

**Review by Adamowoc — DISMISSED** [review]

File: Stormshield/stormshield-endpoint-security/ingest/parser.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          process.parent.executable: "{{json_event.message.SpecificData.ParentProcess.ProcessImageName}}"
+          process.parent.command_line: "{{json_event.message.SpecificData.ParentProcess.ProcessCommandLine}}"
+          process.parent.hash.sha1: "{{json_event.message.SpecificData.ParentProcess.HashSha1}}"
+          process.parent.hash.md5: "{{json_event.message.SpecificData.ParentProcess.HashMd5}}"
+          process.parent.hash.sha256: "{{json_event.message.SpecificData.ParentProcess.HashSha256}}"
+          process.parent.user.id: "{{json_event.message.SpecificData.ParentProcess.User}}"
+          stormshield.process.parent.user.domain: "{{json_event.message.SpecificData.ParentProcess.UserDomainLookup}}"
+          process.parent.user.name: "{{json_event.message.SpecificData.ParentProcess.UserNameLookup}}"
+
+          file.owner: "{{json_event.message.SpecificData.FileOwnerNameLookupp}}"
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply Stormshield/stormshield-endpoint-security/ingest/parser.yml:51]: You are right, in the doc it's `Lookupp`, but in samples it's `Lookup`

File: Stormshield/stormshield-endpoint-security/ingest/parser.yml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+          process.pid: "{{json_event.message.SpecificData.SourceProcess.PID}}"
+          process.start: "{{json_event.message.SpecificData.SourceProcess.ProcessStartTime}}"
+          process.executable: "{{json_event.message.SpecificData.SourceProcess.ProcessImageName}}"
+          process.command_line: "{{json_event.message.SpecificData.SourceProcess.ProcessCommandLine}}"
+          process.hash.sha1: "{{json_event.message.SpecificData.SourceProcess.HashSha1}}"
+          process.hash.md5: "{{json_event.message.SpecificData.SourceProcess.HashMd5}}"
+          process.hash.sha256: "{{json_event.message.SpecificData.SourceProcess.HashSha256}}"
+          process.user.id: "{{json_event.message.SpecificData.SourceProcess.User}}"
+          stormshield.process.user.domain: "{{json_event.message.SpecificData.SourceProcess.UserDomainLookup}}"
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply Stormshield/stormshield-endpoint-security/ingest/parser.yml:45]: `process.user.domain` didn't work I had a taxonomy error even if this's the corresponding field in the ECS documentation.

File: Stormshield/stormshield-endpoint-security/_meta/fields.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+stormshield.specificData.targetProcess.processcommandline:
+  type: keyword
+  description: stormshield process command line
+  name: stormshield.specificData.targetProcess.processcommandline
+
+stormshield.process.user.domain:
+  type: keyword
+  description: Name of the directory the user is a member of
+  name: stormshield.process.user.domain
```
Comment (squioc) [inline Stormshield/stormshield-endpoint-security/_meta/fields.yml:14]: suggest using `stormshield.ses` as prefix:
 ```suggestion
stormshield.ses.process.user.domain:
  type: keyword
  description: Name of the directory the user is a member of
  name: stormshield.ses.process.user.domain
```

File: Stormshield/stormshield-endpoint-security/_meta/fields.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+stormshield.process.parent.user.domain:
+  type: keyword
+  description: User's domain associated with the parent process of the event
+  name: stormshield.process.parent.user.domain
```
Comment (squioc) [inline Stormshield/stormshield-endpoint-security/_meta/fields.yml:19]: I suggest using `stormshield.ses` as prefix
 ```suggestion
stormshield.ses.process.parent.user.domain:
  type: keyword
  description: User's domain associated with the parent process of the event
  name: stormshield.ses.process.parent.user.domain
```

File: Stormshield/stormshield-endpoint-security/_meta/fields.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+stormshield.type:
+  type: keyword
+  description: Event Type ( it's a number )
+  name: stormshield.type
```
Comment (squioc) [inline Stormshield/stormshield-endpoint-security/_meta/fields.yml:24]: I suggest using `stormshield.ses` as prefix:
```suggestion
stormshield.ses.type:
  type: keyword
  description: Event Type ( it's a number )
  name: stormshield.ses.type
```

File: Stormshield/stormshield-endpoint-security/_meta/fields.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+stormshield.additionaldata.incidentguid:
+  type: keyword
+  description: stormshield incident guid
+  name: stormshield.additionaldata.incidentguid
+
+stormshield.specificData.targetProcess.processcommandline:
+  type: keyword
+  description: stormshield process command line
+  name: stormshield.specificData.targetProcess.processcommandline
```
Comment (squioc) [inline Stormshield/stormshield-endpoint-security/_meta/fields.yml:9]: I suggest using `stormshield.ses` as prefix and `process.target.command_line`  for the name of the :
```suggestion
stormshield.ses.process.target.command_line:
  type: keyword
  description: stormshield process command line
  name: stormshield.ses.process.target.command_line
```

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the work.

Some changes requested.

**Review by squioc — APPROVED** [review]

**Pull request merged**